src/olympia/users/management/commands/migrate_user_photos.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia.users.models import UserProfile
+
+
+log = olympia.core.logger.getLogger('z.users.migrate_user_photos')
+
+
+class Command(BaseCommand):
+    help = 'Migrate user photos to new directory structure and remove orphaned ones'
+
+    # Note: because avatars are only displayed on developer profiles and
+    # heavily cached, we're directly migrating in-place without
+    # backwards-compatibility - UserProfile.picture_dir has already been
```

Comment:
I think there is a word missing from this line of the comment. Should it say "has already been *moved*"?

Reply:
indeed (well, technically it shouldn't say "moved", because we're talking about the implementation here), reworded in 2922dff8e021cd7889594804e34632ab5ca01871